ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -43,6 +43,34 @@ def AddV2(*, x, y, name="AddV2"):
     return ivy.addv2(tf_frontend.to_tensor(x), tf_frontend.to_tensor(y))
 
 
+@with_unsupported_dtypes(
+    {
+        "2.10.0 and below": (
+            "float16"
+            "bool",
```

Comment:
only `float16` and `bool` is necessary here, the others may be removed. please also add `bfloat16`, the tests should be passing after that, but please verify it again locally!